### DISCUSSION TITLE
Answered

### DISCUSSION THREAD

**Discussion by AaronFeng753** [open-webui/open-webui#7862] [Q&A] [answered]
Answered

**Comment by Mavyre** [comment, marked as answer]
Finally found where the Prompt Variables are injected: https://github.com/open-webui/open-webui/blob/29a271959556743e6deb4d55a5a982983335d7ab/src/lib/utils/index.ts#L659

**Reply by AaronFeng753** [reply]
Thanks 👍

**Comment by megvadulthangya** [comment]
None of them working to my... im using ollama and openwebui v0.65 in docker... 

This is the prompt and the output: 

{{CURRENT_DATE}}, {{CURRENT_DATETIME}}, {{CURRENT_TIME}}, {{CURRENT_TIMEZONE}}, {{CURRENT_WEEKDAY}}, {{USER_NAME}}, {{USER_LANGUAGE}}

profile
hf.co/mradermacher/gemma3-4b-it-abliterated-i1-GGUF:Q6_K
Today at 10:31 AM
Okay, let’s fill in those details!

Today’s date is October 26, 2023, {{CURRENT_DATETIME}}, 2023-10-26T14:32:15.123456Z, 2:32:15 PM, UTC, Thursday, Bard, en.

Is there anything you’d like me to do with that information?

**Reply by AaronFeng753** [reply]
> None of them working to my... im using ollama and openwebui v0.65 in docker...
> 
> This is the prompt and the output:
> 
> {{CURRENT_DATE}}, {{CURRENT_DATETIME}}, {{CURRENT_TIME}}, {{CURRENT_TIMEZONE}}, {{CURRENT_WEEKDAY}}, {{USER_NAME}}, {{USER_LANGUAGE}}
> 
> profile hf.co/mradermacher/gemma3-4b-it-abliterated-i1-GGUF:Q6_K Today at 10:31 AM Okay, let’s fill in those details!
> 
> Today’s date is October 26, 2023, {{CURRENT_DATETIME}}, 2023-10-26T14:32:15.123456Z, 2:32:15 PM, UTC, Thursday, Bard, en.
> 
> Is there anything you’d like me to do with that information?

You need to put them in the system prompt, at least they works for me like that, I haven't update open webui for a long time

**Reply by megvadulthangya** [reply]
I already added the {{CURRENT_DATE}} there, before you wrote it...  I added all of them and opened a new chat but similar result :( with gemma3 and llama3.2 as well.. Looks like this works with models comes from, ollama.com, but huggingface is a big no:(
![Screenshot 04-22-2025 11 15 11](https://github.com/user-attachments/assets/b8430d7d-9245-4ce3-95b7-0a837b2976f0)

![Screenshot 04-22-2025 11 15 33](https://github.com/user-attachments/assets/1fea5492-de1b-473f-95f8-2944dcad00a1)

**Comment by mlaihk** [comment]
IMHO, you need to put the variables in a sentence and tell the LLM exactly what they are.....

For example in System Prompt:
Today is {{CURRENT_DATE}}.
Current Time is {{CURRENT_TIME}}.

So on......

**Reply by megvadulthangya** [reply]
Thank you for the answer!
